### DISCUSSION TITLE
Keyfilter is not working on mobile

### DISCUSSION THREAD

**Discussion by leo-ay** [primefaces/community#2917] [PrimeVue]
Tested on the component page docs itself. 
https://primevue.org/keyfilter/

with Android with Brave and Chrome Browsers.

Can somebody else confirm?

PrimeReact seems to NOT working too.

PrimeNG seems to be working correctly.

**Comment by j0rgedev** [comment]
It works with iOS Chrome app

**Reply by leo-ay** [reply]
Thanks for testing. But here it still not :(

**Comment by leo-ay** [comment]
Recorded a video.

https://github.com/user-attachments/assets/79401db8-4399-43ba-83c0-6c9b56b55412

**Comment by Counterworks-Nick** [comment]
KeyFilter isn't working for our web app on version ^17.18.6. Our use case is an input field inside a p-cellEditor. As such (pardon the tailwind collapse squiggles):
![image](https://github.com/user-attachments/assets/84836b59-c6a7-4dd9-a123-cc5c1e969f44)
![image](https://github.com/user-attachments/assets/7e3943dd-bd3b-4a27-b2dc-222c84b2b3e8)

It also doesnt work [here ](https://stackblitz.com/edit/584xp4?file=package.json) in their stackblitz example
